pyproject.toml

```diff
@@ -362,7 +362,8 @@ filterwarnings = [
   # https://github.com/pydata/xarray/pull/8939
   "ignore:__array__ implementation doesn't accept a copy keyword, so passing copy=False failed.",
   "ignore:pkg_resources is deprecated",
-  "ignore:Deprecated call to `pkg_resources.declare_namespace"
+  "ignore:Deprecated call to `pkg_resources.declare_namespace",
+  "ignore:Use Group.create_array instead."
```

Comment:
Should this actually be ignored...? Maybe add a comment linking the relevant lines of code and any update guides?

Reply:
In this PR I try to make zarr test passing again, I do not fully understanad the test cases. 
```
napari_builtins/_tests/test_io.py:80 test_zarr_nested - DeprecationWarning: Use Group.create_array instead.…
napari_builtins/_tests/test_io.py F                                                                                                                                [ 4/83]
napari_builtins/_tests/test_io.py:91 test_zarr_with_unrelated_file - DeprecationWarning: Use Group.create_array instead.…
napari_builtins/_tests/test_io.py F                                                                                                                                [ 5/83]
napari_builtins/_tests/test_io.py:105 test_zarr_multiscale - DeprecationWarning: Use Group.create_array instead.…
```

Reply:
Digging into zarr a bit, `create_dataset` was just for h5py compatibility.
See: https://github.com/zarr-developers/zarr-python/blob/a9d6d74c4a4d4efcdb64015919d0a5685a7e0239/src/zarr/core/group.py#L1630-L1646

And: https://github.com/zarr-developers/zarr-python/pull/571

As this is just in our tests, I think we can just replace `create_dataset` with `create_array` with no issues.

Reply:
I made that as a suggestion.